python/amico/core.py

```diff
@@ -51,8 +52,9 @@ def __init__( self, study_path, subject ) :
 
         self.set_config('peaks_filename', None)
         self.set_config('doNormalizeSignal', True)
+        self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
-
+        self.set_config('doSaveCorrectedDWI', False)
```

Comment:
I like the idea to save some extra data like e.g. the predicted dMRI signal or the estimated coefficients. I thinbk we just need to find a way to be more general, as every model might need special care.